src/orchard/directory.clj

```diff
@@ -0,0 +1,67 @@
+(ns orchard.directory
```

Comment:
I think this namespace should be named in some more generic way (provided we want to keep there os-related functionality). Maybe `orchard.os` or `orchard.platform`? Not sure how this is typically named.

Reply:
I like `orchard.os` shorter!